{
    "timestamp": 1556942964892,
    "status": 500,
    "error": "Internal Server Error",
    "exception": "org.springframework.dao.DuplicateKeyException",
    "message": "\n### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '2000' for key 'PRIMARY'\n### The error may involve ProductMapper.insert-Inline\n### The error occurred while setting parameters\n### SQL: insert into products values(?,?,?)\n### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '2000' for key 'PRIMARY'\n; SQL []; Duplicate entry '2000' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '2000' for key 'PRIMARY'",
    "path": "/product"
}



{
    "msg": "No bean named 'asyncPoolTaskExecutor' available: No matching Executor bean found for qualifier 'asyncPoolTaskExecutor' - neither qualifier match nor bean name match!"
}

{
    "msg": "org.springframework.util.concurrent.ListenableFutureTask cannot be cast to org.springframework.scheduling.annotation.AsyncResult"
}

